woocommerce/woocommerce-gateway-stripe
# Use correct class name for Link when disabling it for Checkout editor page

This is a follow up to https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3516 in which a refactor I made at last moment made that PR useless. This PR fixes that.


## Testing instructions

1. Ensure Link is working fine.
2. In Checkout editor page, Link shouldn't show as not compatible.
